shared/studio/components/sessionState/sessionState.module.scss

```diff
@@ -140,6 +140,9 @@
   pointer-events: none;
   transition: opacity 0.2s;
 
+  @include breakpoint(sm) {
+    display: none;
+  }
```

Comment:
I guess this is a stray part of the client settings update?

Reply:
I removed it because I thought we will show client settings differently on mobile, I didn't see the bar on any of mobile screens. U think it will be the same as on desktop and we should leave it?

Reply:
No, I thought this was part of the client settings work?

Reply:
well that's true, this is also in the client settings PR, but if we merge this screen firstly I want to remove it here too so the screen looks like in Figma